app-office/worklog-assistant-bin/worklog-assistant-bin-2.1.3577.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit unpacker
+
+DESCRIPTION="Hassle-free time tracking for JIRA self-hosted and OnDemand"
+HOMEPAGE="https://worklogassistant.com"
+
+MY_PN="${PN%-bin}"
+SRC_URI="https://worklogassistant.com/downloads/${MY_PN}-v2_${PV}_amd64.deb"
+
+LICENSE="worklog-assistant"
+SLOT="0"
+KEYWORDS="~amd64"
+
```

Comment:
I suggest adding empty `IUSE=""` to save others from looking if it's somewhere else.